### ISSUE TITLE
Add Emotes / Character Requests (3)

### ISSUE THREAD

**Issue by objection-lol** [objection-lol/website#153]
### Please use this pin to request adding emotes for existing characters or request adding a new character.

**Comment by LLightNiNGMoNSTeR** [comment]
Hey, can you please add Mia's theme (Turnabout Sisters' Ballad)?

**Comment by aladairmaxwell** [comment]
HD gavel for Judge

**Comment by LordFawful** [comment]
Gallery from AJ

**Comment by aladairmaxwell** [comment]
Wendy Oldbag in alien suit from JFA

**Comment by PavelChitaylo** [comment]
https://www.spriters-resource.com/3ds/phoenixwrightaceattorneytrilogy/sheet/67096/
Dick Gumshoe with a bandage head from Farewell, My Turnabout and with a beige coat from Turnabout Beginnings

**Comment by aladairmaxwell** [comment]
Adrian Andrews from T&T

**Comment by Hanif2108** [comment]
Apollo Justice's gallery and defendant lobby

**Comment by Misha1205** [comment]
I know, that this not emote or character, but i'd like to see the Search ~ Core 2002 in objection.lol. I have a file!
And i have file for Hotline of Fate too.

**Comment by RichardWell56** [comment]
Hi!
I upgrade my Super-Objection animation:

<details>
<summary>New Super-Objection</summary>

![imgonline-com-ua-GIF-Animation-zR0cl6pdvJx13](https://user-images.githubusercontent.com/73120927/102125267-15dd5800-3e52-11eb-8b2a-f2cf236beec3.gif)
![imgonline-com-ua-GIF-Animation-f6fe9YQKY8njyS](https://user-images.githubusercontent.com/73120927/102125280-1d046600-3e52-11eb-8045-b70d09893ec4.gif)

**Comment by Misha1205** [comment]
> Hi!
> I upgrade my Super-Objection animation:
> 
> New Super-Objection
> ![imgonline-com-ua-GIF-Animation-zR0cl6pdvJx13](https://user-images.githubusercontent.com/73120927/102125267-15dd5800-3e52-11eb-8b2a-f2cf236beec3.gif)
> ![imgonline-com-ua-GIF-Animation-f6fe9YQKY8njyS](https://user-images.githubusercontent.com/73120927/102125280-1d046600-3e52-11eb-8045-b70d09893ec4.gif)

Oh, i like this version. its cool

**Comment by RichardWell56** [comment]
> I know, that this not emote or character, but i'd like to see the Search ~ Core 2002 in objection.lol. I have a file!
> And i have file for Hotline of Fate too. Now i have it in customs.

Here the tags: Hof [#bgm8420], Core2002 [#bgm5487]

**Comment by Misha1205** [comment]
> > I know, that this not emote or character, but i'd like to see the Search ~ Core 2002 in objection.lol. I have a file!
> > And i have file for Hotline of Fate too. Now i have it in customs.
> 
> Here the tags: Hof [#bgm8420], Core2002 [#bgm5487]

Yes, i know. I have it in customs but id like to see it how preset.

**Comment by Hanif2108** [comment]
I don't know, maybe Gyakuten Kenji 2/AAI characters with its background?

**Comment by objection-lol** [comment]
@RichardWell56 That's a good one, though I'd like to ask how did you save the GIF? Because there are "jumping dust particles" on Phoenix when the animation is playing:
![image](https://user-images.githubusercontent.com/71948517/102171781-5ebffb80-3ea8-11eb-8a2f-a5565d609804.png)
If you have used Photoshop then you can give me the Photoshop source file.

**Comment by Misha1205** [comment]
> I don't know, maybe Gyakuten Kenji 2/AAI characters with its background?

No. Because objection.lol based on AA1-4 games, and yo can add it for custom characters and backgrounds now

**Comment by RichardWell56** [comment]
> @ RichardWell56 Это хороший вопрос, хотя я хотел бы спросить, как вы сохранили GIF? Поскольку при воспроизведении анимации на Phoenix появляются «прыгающие частицы пыли»: если вы использовали Photoshop, вы можете предоставить мне исходный файл Photoshop.
> ![образ](https://user-images.githubusercontent.com/71948517/102171781-5ebffb80-3ea8-11eb-8a2f-a5565d609804.png)

Sorry, but I didn’t make this GIF in Photoshop, but on the site in Google
